conda/recipes/numbast/meta.yaml

```
@@ -20,32 +20,25 @@ build:
   script:
     - {{ PYTHON }} -m pip install . -vv
   ignore_run_exports_from:
-    - {{ compiler('cuda') }}
     - libnvjitlink-dev
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}
     - cmake >=3.28
-    - pybind11
-    - cffi
-    - clangdev >=18
     - ninja
     - ast_canopy
     - sysroot_{{ target_platform }} 2.17
   host:
-    - cuda-version {{ cuda_compiler_version }}
     - python
     - pip
-    - scikit-build-core
   run:
     - python
     - numba >=0.58
     - pynvjitlink >=0.2
+    - cuda-nvrtc {{ cuda_compiler_version }}
```

Don't pin CUDA packages directly. Use `cuda-version`. But do we want a `>=` here? Maybe we should put `cuda-version` in `host` and use `pin_compatible` as we do for RAPIDS. Example: https://github.com/rapidsai/cudf/blob/cdfb550f442e846623c721082128a095f02efff9/conda/recipes/libcudf/meta.yaml#L92-L96
```suggestion
    - cuda-version {{ cuda_compiler_version }}
    - cuda-nvrtc
```